instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/utils.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    # Utility functions
+    module Utils
```

Comment:
The utils here are shameless ported from some existing instrumentation, there is some room for feedback/improvement here.

Reply:
https://github.com/open-telemetry/opentelemetry-ruby/issues/339 this is also feels very relevant here

Reply:
Seems fine to me, i don't have strong opinions on the default placeholder but that's the only thing maybe we may want to change. not a big deal at all, ship it imo